source/developers.markdown

```diff
@@ -34,7 +34,7 @@ title: Developer Resources for Apache CloudStack
 
 <p>Apache CloudStack has a read-only mirror on <a href="https://github.com/apache/cloudstack" target="_blank">GitHub</a> that is kept in sync with the canonical Git repo maintained by the Apache Software Foundation. Submitting GitHub pull requests is the easiest way to get your contribution upstream. For detailed instructions see the link below:</p>
 
-<br><a href="https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md" target="_blank">GitHub Contribution Guidelines</a></br>
+<br><a href="https://github.com/apache/cloudstack/blob/master/CONTRIBUTING.md" target="_blank">GitHub Contribution Guidelines</a>
```

Comment:
@karuturu yes it is. I removed because the visual effect with `<br/>` and without looked the same (at least in my eyes) ;)

I do not have the best sight for theses things; I can go back and se the `<br/>`.
What do you think?